lib/taskana-core-test/src/test/java/acceptance/taskcomment/delete/DeleteTaskCommentAccTest.java

```diff
@@ -0,0 +1,170 @@
+
+import java.time.Instant;
+import java.util.List;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestTemplate;
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.Classification;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.exceptions.InvalidArgumentException;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.exceptions.NotAuthorizedOnTaskCommentException;
+import pro.taskana.task.api.exceptions.TaskCommentNotFoundException;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.api.models.TaskComment;
+import pro.taskana.testapi.DefaultTestEntities;
+import pro.taskana.testapi.TaskanaInject;
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.builder.TaskBuilder;
+import pro.taskana.testapi.builder.TaskCommentBuilder;
+import pro.taskana.testapi.builder.WorkbasketAccessItemBuilder;
+import pro.taskana.testapi.security.WithAccessId;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.Workbasket;
+
+@TaskanaIntegrationTest
+class DeleteTaskCommentAccTest {
+
+  private static final String ACCESS_USER_1_1_ID = "user-1-1";
+
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+  @TaskanaInject TaskanaEngine taskanaEngine;
+
+  Classification defaultClassification;
+  Workbasket defaultWorkbasket;
+  Task task1;
+  TaskComment comment1;
+  TaskComment comment2;
+
+  @WithAccessId(user = ACCESS_USER_1_1_ID)
+  @BeforeEach
+  public void createTaskAndTaskComments() throws Exception {
+    task1 =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassification.asSummary())
+            .workbasketSummary(defaultWorkbasket.asSummary())
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+    comment1 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    comment2 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    List<TaskComment> taskComments = taskService.getTaskComments(task1.getId());
+    assertThat(taskComments).hasSize(2);
+  }
+
+  @WithAccessId(user = "admin")
+  @AfterEach
+  public void deleteTask() throws Exception {
```

Comment:
If we delete the previous method, we won't need this one too

Reply:
Yes I have removed it